CefSharp.Wpf.Example/Views/BrowserTabView.xaml

```diff
@@ -168,12 +168,15 @@
                     </Rectangle.Fill>
                 </Rectangle>
 
+                    <ContentControl Content="{Binding WebBrowser}"
+                                Opacity="{Binding ElementName=opacitySlider, Path=Value}" >
+                    <!--
                 <cefSharp:ChromiumWebBrowser x:Name="browser"
```

Comment:
I completely agree - it's not nice. But I could find no other way of delay-loading the control instance, and conform the the MVVM pattern to some degree. I'm not really happy with how the ChromiumWebBrowser gets created first, then added to the visual tree later - it seems the underlying control should probably manage that.